reflex/components/radix/themes/components/radio_group.py

```diff
@@ -146,7 +146,7 @@ def create(
         # convert only non-strings to json(JSON.stringify) so quotes are not rendered
         # for string literal types.
         if (
-            type(default_value) is str
+            type(default_value) is str  # noqa: E721
```

Comment:
why not use `isinstance` here like in other files?
if there is a specific reason we should add a comment why.

Reply:
i had some problem recently with bool/int and was not sure if sudden failing test was related...